bitnami/jenkins/templates/deployment.yaml

```diff
@@ -16,6 +16,9 @@ spec:
   {{- $podLabels := include "common.tplvalues.merge" ( dict "values" ( list .Values.podLabels .Values.commonLabels ) "context" . ) }}
   selector:
     matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" $podLabels "context" $ ) | nindent 6 }}
+  {{- if .Values.replicaCount }}
```

Comment:
Change it to this solution: https://github.com/bitnami/charts/blob/main/bitnami/apache/templates/deployment.yaml#L19

Reply:
Fixed